pages/linux/fuser.md

```diff
@@ -6,3 +6,19 @@
 - Identify process using a TCP socket:
 
 `fuser -n tcp {{port}}`
+
+- Find which process accessing a directory:
```

Comment:
Perhaps this will read better?
```suggestion
- Find which process is accessing a directory:
```

Also, could `path_to_dir` be `path/to/directory` please? It's more consistent with other pages that way :-) The same goes for the instances below too.

Reply:
What if there are more than one process accessing a directory ?

Reply:
Hi @sbrl and @agnivade I've improved the text to address this issues.

Reply:
Hi @mendesmiguel - I still think it is not clear. My question above does not seem to be answered. Your description still says "Find which process"